src/main/resources/drivers/Huawei_NE.js

```diff
@@ -288,7 +288,7 @@ function snapshot(cli, device, config) {
 		device.add("vrf", match[1]);
 	}
 	
-	var interfaces = cli.findSections(currentConfig, /^interface (.+)/m);
+	var interfaces = cli.findSections(currentConfig, /^interface (.+\.[0-9]+)/m);
```

Comment:
Physical interfaces wouldn't be matched anymore...
`/^interface ([^ ]+)( .+)?/m` might make more sense, what do you think?

Reply:
Agreed. this looks better. regex is not my strongpoint :-)